### PR TITLE
chore(angular): update angular to v9

### PR DESCRIPTION
**Pull request by conundrumgirl**
attempt to upgrade to angular 9

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -253,7 +251,7 @@
     "url": "https://github.com/Sage-Bionetworks/Agora/issues"
   },
   "engines": {
-    "node": ">= 8.0.0",
+    "node": ">= 10.13.0",
```
Comment (zaro0508) [inline package.json:257]: if it help we are currently running node 12.14.1 and npm 6.13.4

**Review by zaro0508 — APPROVED** [review]

**Comment by zaro0508** [comment]
ohh but build failed

**Comment by conundrumgirl** [comment]
@zaro0508  - fixed it:) OK to merge?

**Review by zaro0508 — APPROVED** [review]
looks good to me

**Pull request merged**